linux/utils/check_service_status.yml

```diff
@@ -29,8 +29,21 @@
     (expected_service_status is undefined or
      expected_service_status not in ['enabled', 'disabled', ''])
 
-- name: "Get service information"
-  include_tasks: get_service_info.yml
+- name: "Retry to get service information"
```

Comment:
`service_info` can be initialized firstly in case it's defined somewhere else. Then below `service_info is defined` check can be removed.